teps/0098-workflows.md

```diff
@@ -141,8 +141,40 @@ leveraging Workflows for some common functionality but still relying on their ow
 In addition to extensibility mechanisms, we should explore creating a Workflows conformance spec. This will allow existing (and new!) CI/CD projects
 to become Workflows conformant if they don't want to use the default Workflows implementation, giving end users more flexibility in where to run their Workflows.
 
+### Use of Triggers
```

Comment:
@alanghelardi @rafaeleal can you comment on how this aligns with your experience with triggers?